src/compiler/parser.cc

```diff
@@ -210,7 +210,10 @@ namespace verona::compiler
     Rule fn_signature = generics >> function_params >> -(":" >> type) >>
       -ExprPtr(where_clauses);
     Rule fn_body = block;
-    Rule method = def_ident >> fn_signature >> (fn_body | ";");
+
+    Rule method_builtin = "builtin"_E;
```

Comment:
Should a builtin have a body? I guess can give better error message if we parse it, and then say why we are unhappy?

Reply:
Yes. I've added a test for the error message.